workspace/workflows/jobs.py

```diff
@@ -141,6 +137,14 @@ def get_latest_conclusions(self) -> dict:
         }
         return conclusions
 
+    @staticmethod
+    def get_conclusion_for_run(run) -> str:
+        aliases = {"in_progress": "running"}
+        if run["conclusion"] is None:
+            status = str(str(run["status"]))
```

Comment:
I think the double string is a typo. As to whether run["status"] is always a string: I am not confident that I have encountered all types of run statuses yet, so would just like to prevent everything from breaking when both run["status"] and run["conclusion"] are null.